bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/FileAudioStream.java

```diff
@@ -102,7 +104,9 @@ public AudioFormat getFormat() {
 
     @Override
     public int read() throws IOException {
-        return inputStream.read();
+        int read = inputStream.read();
+        alreadyRead++;
```

Comment:
Isn't it rather `alreadyRead += read;` ?

Reply:
The "alreadyRead" counter is the amount of bytes read.
The read() method reads only one byte and return the value of this byte.

Reply:
Ok